Source/Particles/MultiParticleContainer.cpp

```diff
@@ -1607,6 +1630,13 @@ void MultiParticleContainer::doQedQuantumSync (int lev,
         pc_source ->defineAllParticleTiles();
         pc_product_phot->defineAllParticleTiles();
 
+        const bool store_chi_at_creation = pc_product_phot->store_chi_at_creation();
+        int chi_at_creation_runtime_comp = 0;
+        if (store_chi_at_creation){
+            chi_at_creation_runtime_comp =
+                                    pc_product_phot->particle_runtime_comps["qs_chi_at_creation"];
+        }
```

Comment:
If you want to make this more compact and if you want to make `chi_at_creation_runtime_comp` a const quantity you can do:
```suggestion
        const int chi_at_creation_runtime_comp =  (store_chi_at_creation)?
            pc_product_phot->particle_runtime_comps["qs_chi_at_creation"] : 0;
```

Reply:
Thanks for the suggestion, I've used this construct wherever possible.